pkg/slack/modals/launch/launch.go

```diff
@@ -8,15 +8,44 @@ import (
 	"github.com/openshift/ci-chat-bot/pkg/slack/modals"
 	"github.com/sirupsen/logrus"
 	"github.com/slack-go/slack"
+	"k8s.io/apimachinery/pkg/util/sets"
+	"k8s.io/klog"
 	"net/http"
 	"strings"
 	"sync"
 )
 
-// Identifier is the view identifier for this modal
-const Identifier modals.Identifier = "launch"
-const Identifier2ndStep modals.Identifier = "launch2ndStep"
-const Identifier3rdStep modals.Identifier = "launch3ddStep"
+const (
+	IdentifierInitialView        modals.Identifier = "launch"
+	Identifier3rdStep            modals.Identifier = "launch3ddStep"
```

Comment:
Should `launch3ddStep` be `launch3rdStep`?

Reply:
Fixed